polynote-runtime/src/main/scala/polynote/runtime/ValueRepr.scala

```diff
@@ -235,15 +262,13 @@ object StreamingDataRepr {
     repr
   }
 
-  def apply(dataType: DataType, knownSize: Int, lazyIter: => Iterator[ByteBuffer]): StreamingDataRepr =
-    apply(dataType, Some(knownSize), lazyIter)
-
-  def apply(dataType: DataType, lazyIter: => Iterator[ByteBuffer]): StreamingDataRepr =
-    apply(dataType, None, lazyIter)
-
 }
 
 sealed trait ValueRepr
 
+sealed trait TableOp
+final case class GroupAgg(columns: List[String], aggregations: List[(String, String)]) extends TableOp
```

Comment:
There could be a case to be made that these aggregations should be a typed enum.